### PR TITLE
Add image ordering functionality

### LINKED ISSUE #84: Image overlay ordering system
This has been marked as a good candidate for becoming a [first-timers-only issue](http://www.firsttimersonly.com) like [these](https://github.com/publiclab/plots2/labels/first-timers-only), meaning that it's simple, self-contained, and with some extra formatting, could be a great entry point for a new contributor. If you're familiar enough with this code, please consider [reformatting or reposting it as a first-timers-only issue](https://publiclab.org/notes/warren/10-31-2016/create-a-welcoming-first-timers-only-issue-to-invite-new-software-contributors), and then ping @jywarren to get it labelled. Or, if this is not your first time, try to solve it yourself!

****

We should implement image ordering -- in Leaflet, order is managed with CSS `z-index`: 

http://leafletjs.com/reference-1.2.0.html#imageoverlay-zindex

Via Leaflet, then, we already have a `.bringToBack()` method and `.bringToFront()` for each layer, including each image. We just need to add a toolbar button for it -- let's start with `.bringToFront()`.

Here's an example of a button and associated functionality in the toolbar:

https://github.com/publiclab/Leaflet.DistortableImage/blob/3714b06bcdae2ff088c1f52c32ec322d120b4962/src/edit/DistortableImage.EditToolbar.js#L12-L25

I believe we can just say `this.bringToFront()` inside a new button function!

We'll need to choose an icon (http://fontawesome.io/icons/), and this should be pretty straightforward with the above -- but I'm happy to offer advice if you want to take this on!

Any help with this would be appreciated!

### PR DESCRIPTION
**Pull request by rexagod**
Fixes #84. Wired ordering to toolbar and "x" key, as shown below.

peek 2019-02-23 18-37

/cc @jywarren

### CODE REVIEW

**Comment by rexagod** [comment]
#126

**Comment by rexagod** [comment]
Adding docs now...

**Comment by jywarren** [comment]
This is awesome! I wonder if we are starting to need submenus here so that we can add "move up" "move down" "move to top" and "move to bottom" -- but what we could do for the time being is to create this function with only keybindings, starting with "higher" and "lower" - and think about the UI in follow-up. What do you think?

**Comment by jywarren** [comment]
Also my apologies, i should stop using the GitHub UI for "resolve" because it does a merge. This gets squashed and cleaned up after, but it's still messy. Feel free to discard the last commit and do it cleanly if you'd like. 👍

File: dist/leaflet.distortableimage.js [hunk 1]
```diff
@@ -699,13 +718,15 @@ L.DistortableImage.Edit = L.Handler.extend({
 			79: '_toggleOutline', // o
 			82: '_toggleRotateDistort', // r
       84: '_toggleTransparency', // t
+			88: '_toggleOrder', // x
```
Comment (jywarren) [inline dist/leaflet.distortableimage.js]: Here we could bind a `bringForward` and `...Back` or equivalent to the up arrow and down arrows?

**Comment by jywarren** [comment]
Reference for Leaflet.toolbar's sub-menus: 

image

**Comment by rexagod** [comment]
@jywarren Maybe we should create a new issue for submenus. Should I open one up?

File: dist/leaflet.distortableimage.js (hunk 1 above)
Comment (rexagod) [reply dist/leaflet.distortableimage.js]: It seems that all the arrows are defaulted to map movement in the leaflet module. Should I override those wirings?

**Comment by jywarren** [comment]
Oh darn. Is it possible to do shift-arrow? Otherwise maybe we default to j
and k for up and down?

**Comment by rexagod** [comment]
Wired `sendUp` and `sendDown` to `j` and `k` keys, respectively. I suppose pressing a single key would be more convenient that `shift+up/down`. Is this alright?

**Comment by jywarren** [comment]
Hi @rexagod - no rush here but I was hoping merging this would simplify my review of #147 and we could get these closed out. Thanks!

**Comment by rexagod** [comment]
Apologies, @jywarren! I was actually a little busy wrapping up the auto-stitcher module for LDI to include in the proposal, do let me know what you think!! :smile: 

**This one below was configured to stitch images together so as to connect the roads in a continuous manner** :tada: 

ezgif-2-ab309657b88f

**Comment by jywarren** [comment]
Oh my gosh! Wow!

**Comment by jywarren** [comment]
I want to know more about how that works!?! 👍

**Pull request merged**